### ISSUE TITLE
Embedded code variations (php, erb, etc.) don't work in fenced-code-blocks

### ISSUE THREAD

**Issue by burodepeper** [burodepeper/language-markdown#77]
See #22 and #43 (both closed as a way of hiding them)

Summary:<br>
The generated fenced-code-blocks for `php` and `erb` (and possibly others in the future) don't work because there is something wrong with the scope. I suspect the issue lies in the fact that a `.php` file has an opening and closing tag to designate which parts are php; the content of the file isn't php by itself! Adding the opening and closing tags to code-examples doesn't work however.

**Comment by btquanto** [comment]
I also experience problems with YAML
![Yaml fenced code block hightlighting error](http://4.1m.yt/STpdkEq.png)

**Comment by burodepeper** [comment]
@btquanto Your issue is different though no less valid. It looks like the YAML parser doesn't "exit", and in doing so doesn't return control over the parsing back to language-markdown. There's an old issue related to language-gfm where this was first reported (see https://github.com/atom/language-gfm/issues/40) and it appears to be an issue with a small number of other language packs. I will verify whether this is indeed an issue with the yaml parser (and not language-markdown) and move the issue there accordingly.

To verify that there's no conflict with other packages, could you paste a list of your installed packages here? `apm list --installed` in your Terminal should do the trick.

**Comment by burodepeper** [comment]
@btquanto Your issue is most likely related to https://github.com/atom/first-mate/issues/83

**Comment by btquanto** [comment]
Hi, thank you for replying, this is the result of `apm list --installed` on my machine

```
Community Packages (3) /home/btquanto/.atom/packages
├── git-time-machine@1.5.4
├── language-markdown@0.18.2
└── platformio-ide-terminal@2.2.2
```

**Comment by burodepeper** [comment]
Verified the issue, and added a response to the issue in atom/first-mate.

> May I ask if there is a way to disable fenced code highlighting for a particular language?

I will publish an update shortly that disables YAML for the time being. YAML seems to cause issues with even the simplest of content, so until that's fixed, I don't see a reason to keep it in.

**Comment by btquanto** [comment]
Thank you :)

**Comment by burodepeper** [comment]
Fixed in v0.22.0 with a little help from @zachflower

**Comment by kylebarron** [comment]
@btquanto the link to the YAML screenshot you posted is no longer broken, so it's impossible to see what problem you were trying to report.
